.template/addons/docker/Dockerfile.tt

```diff
@@ -38,7 +38,8 @@ RUN apt-get update -qq && \
 ADD https://dl.yarnpkg.com/debian/pubkey.gpg /tmp/yarn-pubkey.gpg
 RUN apt-key add /tmp/yarn-pubkey.gpg && rm /tmp/yarn-pubkey.gpg && \
     echo "deb http://dl.yarnpkg.com/debian/ stable main" > /etc/apt/sources.list.d/yarn.list && \
-    curl -sL https://deb.nodesource.com/setup_"$NODE_SOURCE_VERSION".x | bash - && \
+    echo 'Package: nodejs\nPin: origin deb.nodesource.com\nPin-Priority: 600' > /etc/apt/preferences.d/nodesource && \
```

Comment:
As Ruby-3.2.2 slim is based on the Debian-12, and Debian 12 provides a newer Node.js package 18.13.0 from the ﻿deb.debian.org repository, so to override the Node.js version, we have to pin the deb.nodesource.com first ([source](https://github.com/nodesource/distributions/issues/1579))